configuration/ignored-service-accounts.md

```diff
@@ -0,0 +1,25 @@
+---
+description: This article explains what Ignored Service Accounts are and how to set them up in SysKit Point.
```

Comment:
Article objašnjava što su Service accounts rađe nego Ignored Service Accounts. 
Isto tako, objašnjava kako ignorirati service account activities u Pointu. Kroz Point se ne mogu setupirati/kreirati novi accounti/useri (za sad)